jerry-core/ecma/base/ecma-globals.h

```diff
@@ -331,6 +331,32 @@ typedef struct ecma_native_pointer_t
   struct ecma_native_pointer_t *next_p; /**< points to the next ecma_native_pointer_t element */
 } ecma_native_pointer_t;
 
+/**
+ * Option bits for ecma_parse_options_t.
+ */
+typedef enum
+{
+  /* ECMA_PARSE_STRICT_MODE = (1 << 0) */
+  /* ECMA_PARSE_MODULE = (1 << 1) */
```

Comment:
This commented code looks kinda weird for me. Is this intentional?

Reply:
I wanted to emphasize that those constants are defined elsewhere. Do you have a better suggestion?

Reply:
How about just a simple comment like:

The first 2 bits are restricted for ...., see ...

Reply:
What about:   `/* bit 0: ECMA_PARSE_STRICT_MODE */` ?